test/runtests.jl

```diff
@@ -64,7 +64,9 @@ if !Sys.iswindows()
 @safetestset "FD ops - opt" begin @time include("Operators/finitedifference/opt.jl") end
 @safetestset "FD ops - wfact" begin @time include("Operators/finitedifference/wfact.jl") end
 @safetestset "FD ops - linsolve" begin @time include("Operators/finitedifference/linsolve.jl") end
-@safetestset "FD ops - examples" begin @time include("Operators/finitedifference/opt_examples.jl") end
+if VERSION >= v"1.7.0"
+    @safetestset "FD ops - examples" begin @time include("Operators/finitedifference/opt_examples.jl") end
+end
```

Comment:
Our current Project.toml requires julia 1.7, so this is redundant.
```suggestion
@safetestset "FD ops - examples" begin @time include("Operators/finitedifference/opt_examples.jl") end
```